### ISSUE TITLE
Border Radius with Color on Container

### ISSUE THREAD

**Issue by iamsankalp** [foundation/foundation-emails#427]
Seems like using border radius and a border with some color on 'container' given via a class is not working as expected. The edges are rounded but the colored border is not.

![image](https://cloud.githubusercontent.com/assets/1756637/15079738/f72dcf92-13d7-11e6-8ee4-eb8c09f33266.png)

**How can we reproduce this bug?**
1) Give a class to container-  let's say: `<container class="content">`
2) `.content{ border-radius: 10px; border: 1px solid red; }
`
**What did you expect to happen?**
Border color along with radius. Like this image: 
![image](https://cloud.githubusercontent.com/assets/1756637/15079819/6d3345b4-13d8-11e6-944b-e20ad53d7e4d.png)

**What happened instead?**
Like the image attached in the very beginning.
**What email clients does this happen in?**
Tested in my browser only so far.

PS: I believe border radius is not supported well across all clients, just that it's presence doesn't break anything so it's fine - to whatever clients it can show up. :)

**Comment by rafibomb** [comment]
This seems like a real challenge with tables. Same happens for buttons. To add a border radius the border needs to be removed. Are you aware of a workaround on this?

**Comment by iamsankalp** [comment]
@rafibomb Yeah, I did find a workaround. Give `border-collapse: separate;` to the same class. Just not sure if it's technically a good way since there are many border-collapse defined. What do you think?

**Comment by rafibomb** [comment]
Good find! Not sure what the implications are for changing that property, but if it passes a Litmus test on Outlook 2000-16 and Gmail it's a good solution. 

@tdhartwick Any thoughts on the border-collapse solution here?

**Comment by rafibomb** [comment]
@iamsankalp Is this something you can add as a PR to the 2.2 branch for this? We can flush it out in testing but I think it's a good solution!

**Comment by rafibomb** [comment]
Hi @iamsankalp Would love to get this into 2.2 coming soon - are you able to submit a PR so we'll test it?

**Comment by iamsankalp** [comment]
@rafibomb - sorry, i missed the previous message completely. I'll try doing a PR.

**Comment by iamsankalp** [comment]
Hey @rafibomb I believe the solution doesn't solve it for Outlook 2010, 2013 as i could check via Litmus.

Holding off the PR for now.

![image](https://cloud.githubusercontent.com/assets/1756637/15870580/5fe3e43c-2d0e-11e6-88c0-76bbe92310db.png)

**Comment by rafibomb** [comment]
Hi @iamsankalp That's ok! We know it wont work for Outlook, maybe just 2016. I have a litmus test here that shows it works for other clients https://litmus.com/checklist/emails/public/3d043d3

Would still love to see your PR and we can collaborate to get it across the line!

**Comment by rafibomb** [comment]
Saw the PR - I was thinking that someone will add a class of `.radius` to the container to add the $container-radius variable to it. Maybe default it to 0.

```
// adds radius to container
table.container.radius {
  border-radius: $container-radius;
  border-collapse: separate;
}
```

Thoughts?

**Comment by iamsankalp** [comment]
Nice solution, @rafibomb!

So, we'll create a variable `$container-radius`. But the class name stays the same - i.e. `radius`. Um... should the class name also be different so that relevance is clear? Plus, the fact that button and container-radius values may not necessarily be same. I'm overthinking maybe.

Sankalp

**Comment by rafibomb** [comment]
In Foundation for Sites, `.radius` is a modifier class that can be added to certain components like callouts and buttons. We can keep it the same and document it

**Comment by rafibomb** [comment]
Ok added it here - https://github.com/zurb/foundation-emails/commit/17b8905bb4e018a4149983f7bd0050a40fe4afb7 Are you able to update the docs?

**Comment by rafibomb** [comment]
I think this is just open to update the docs! @iamsankalp Are you able to add it for the release next Thursday?

**Comment by DanielRuf** [comment]
Any updates on this?

**Comment by joeworkman** [comment]
Foundation for Emails has been superseded by [Inky v2](https://github.com/foundation/inky), a complete rewrite in Rust with improved Outlook support, hybrid output mode, bulletproof buttons, validation, and more. This issue will not be addressed in the legacy codebase.

If this issue persists in v2, please open a new issue at https://github.com/foundation/inky/issues
